### ISSUE TITLE
Telegram Kafka Connect

### ISSUE THREAD

**Issue by IHengki95** [confluentinc/demo-scene#121]
I've tried your demo and it works really great.
I want to explore more about the real-time notification, which you've still put in TODO.
But I see there's "delay_alert" on "egress" so I've tried to figure out how to make the topics until we could create a connector.

I've tried to create all of what I need like the location, delay, the train, and so on into a single markdown to single topic TRAIN_DELAY_ALERT_TG.
The problem is when I create the sink using "01_create_sinks.sh" on the "delay_alert".
here is the error and I don't know what is wrong with the configuration from the sink that you've provided.
![image](https://user-images.githubusercontent.com/26961718/80469633-05cc4300-896b-11ea-820c-a5bf4a633a11.png)

I've put the key for the bot on credential.properties and enter the chat ID.
All is well if I'm using curl to send the message.
Here are the Topic and the value
![image](https://user-images.githubusercontent.com/26961718/80469869-60fe3580-896b-11ea-93e2-22a362451d50.png)

Here are the "01_create_sinks.sh"
![image](https://user-images.githubusercontent.com/26961718/80470064-aa4e8500-896b-11ea-865b-74dd433d473e.png)

**Comment by rmoff** [comment]
The error is about payload being sent to Kafka Connect, and looks like an escaping issue. At a guess, maybe here https://github.com/confluentinc/demo-scene/blob/master/rail-data-streaming-pipeline/data/egress/delay_alerts/01_create_sinks.sh#L12 or here https://github.com/confluentinc/demo-scene/blob/master/rail-data-streaming-pipeline/data/egress/delay_alerts/01_create_sinks.sh#L15 ?

It's a while since I've looked at this. You could put the JSON in a separate file and pass it as an argument to `curl` to make it easier to debug (once less layer of escaping needed). Report back if you get it working!

**Comment by IHengki95** [comment]
Hi @rmoff ,
yes actually I'm trying to use my microservices to push the data from this topic to the Telegram.
And we got the same issue about the blob.

Actually the thing is we can't consume the data because of the blob data.
Regarding the concat to markdown the message.
Have you ever had this error before?

We've tried with multi-field of the value and single field of value to ingest the data and it works.
But when we consume from the concat data using the KSQL, we got the error for blob.
